lib/cli/index.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('../utilities/will-interrupt-process').capture(process);
```

Comment:
lets leave a comment saying why this needs to be at the top of the file (similar to the one we have for `setMaxListeners`.